docs/source/nlp/punctuation_and_capitalization.rst

```diff
@@ -158,55 +162,702 @@ for the training and evaluation in the :code:`conf/punctuation_capitalization_co
+`here <https://github.com/NVIDIA/NeMo/blob/stable/examples/nlp/token_classification/conf/punctuation_capitalization_config.yaml>`_.
+
+.. list-table:: Run config. The main config passed to scripts `punctuation_capitalization_train.py <https://github.com/NVIDIA/NeMo/blob/stable/examples/nlp/token_classification/punctuation_capitalization_train.py>`_ and `punctuation_capitalization_evaluate.py <https://github.com/NVIDIA/NeMo/blob/stable/examples/nlp/token_classification/punctuation_capitalization_evaluate.py>`_
+   :widths: 5 5 10 25
+   :header-rows: 1
+
+   * - **Parameter**
+     - **Data type**
+     - **Default value**
+     - **Description**
+   * - **pretrained_model**
+     - string
+     - ``null``
+     - Can be an NVIDIA's NGC cloud model or a path to a ``.nemo`` checkpoint. You can get list of possible cloud options
+       by calling a method :py:meth:`~nemo.collections.nlp.models.PunctuationCapitalizationModel.list_available_models`.
+   * - **name**
+     - string
+     - ``'Punctuation_and_Capitalization'``
+     - A name of the model. Used for naming output directories and ``.nemo`` checkpoints.
+   * - **do_training**
+     - bool
+     - ``true``
+     - Whether to perform training of the model.
+   * - **do_testing**
+     - bool
+     - ``false``
+     - Whether ot perform testing of the model after training.
+   * - **model**
+     - :ref:`model config<model-config-label>`
+     - :ref:`model config<model-config-label>`
+     - A configuration for the :class:`~nemo.collections.nlp.models.PunctuationCapitalizationModel`.
+   * - **trainer**
+     - trainer config
+     -
+     - Parameters of
+       `pytorch_lightning.Trainer <https://pytorch-lightning.readthedocs.io/en/latest/common/trainer.html#trainer-class-api>`_.
+   * - **exp_manager**
+     - exp manager config
+     -
+     - A configuration with various NeMo training options such as output directories, resuming from checkpoint,
+       tensorboard and W&B logging, and so on. For possible options see :ref:`exp-manager-label` description and class
+       :class:`~nemo.utils.exp_manager.exp_manager`.
+
+.. _model-config-label:
+
+Model config
+^^^^^^^^^^^^
+
+.. list-table:: Location of model config in parent config
+   :widths: 5 5
+   :header-rows: 1
+
+   * - **Parent config**
+     - **Key in parent config**
+   * - :ref:`Run config<run-config-label>`
+     - ``model``
+
+A configuration of
+:class:`~nemo.collections.nlp.models.token_classification.punctuation_capitalization_model.PunctuationCapitalizationModel`
+model.
+
+.. list-table:: Model config
+   :widths: 5 5 10 25
+   :header-rows: 1
+
+   * - **Parameter**
+     - **Data type**
+     - **Default value**
+     - **Description**
+   * - **class_labels**
+     - :ref:`class labels config<class-labels-config-label>`
+     - :ref:`class labels config<class-labels-config-label>`
+     - A mandatory parameter containing a dictionary with names of label id files used in ``.nemo`` checkpoints. These
```

Comment:
I'd reword this part. Users don't need to pass anything to this args, but they're required to create a proper .nemo file with all artifacts. Or do you mean smth different?